config/ftbquests/quests/chapters/refined_storage.snbt

```diff
@@ -280,96 +339,147 @@
 			}]
 		}
 		{
-			x: 0.0d
-			y: -10.5d
+			x: 2.0d
+			y: -8.5d
 			subtitle: "Automated crafting!"
-			description: ["The Crafter is a block that crafts items with patterns. It can take speed upgrades to increase the crafting speed."]
-			dependencies: ["59F5ED931FD70C55"]
+			description: [
+				"The &9Crafter&r is how we store patterns for our Network to know how to craft items. Once a recipe is placed into the crafter, you'll be able to \"Request\" an item to be crafted from your Grid. You can do this by hitting CTRL+Shift when clicking on the item you have a recipe for to bring up the crafting GUI."
+				""
+				"This block can also be placed facing a machine to use &6Processing Patterns&r. For example, if you point the Crafter at a Furnace, you can put a recipe inside for a recipe you need a furnace for, like smelting raw processors. As long as the crafter is attached to your system and the Furnace has an importer attached to it, the Crafter will allow you to request the smelted version of the Processor!"
+			]
+			dependencies: ["65C8A43FEDBA3835"]
+			size: 1.25d
+			min_width: 300
 			id: "44881E39AB1D62AA"
 			tasks: [{
 				id: "202A7EA0318662C5"
 				type: "item"
 				item: "refinedstorage:crafter"
 			}]
-			rewards: [{
-				id: "3351B9B6898579ED"
-				type: "item"
-				item: "refinedstorage:pattern"
-			}]
+			rewards: [
+				{
+					id: "3351B9B6898579ED"
+					type: "item"
+					item: "refinedstorage:pattern"
+					count: 4
+					random_bonus: 4
+				}
+				{
+					id: "6831B9D1AB4346E0"
+					type: "random"
+					table_id: 3567941291661635734L
+				}
+			]
 		}
 		{
 			x: 1.5d
-			y: -10.5d
+			y: -8.0d
+			description: ["An upgraded crafter that holds more patterns and has an increased crafting speed."]
```

Comment:
I would like to mention that there are new speed limitations regarding the Iron, Gold, Diamond and Netherite Crafters: If the crafter is pointed at a container, its crafting speed is limited based on the number of slots in the container.

For example, a crafter pointed at a furnace can only go up to 1x speed, unless the furnace is made a Factory, in which case the maximum speed is up to 6x. A Mekanism Ultimate Factory limits any RS crafters attached to 9x speed.

If you want to go faster, the crafter must be attached to a chest whose items are input into the processing setup.

The original Refined Storage Crafter (and likely the Creative Crafter) is unaffected by this limitation.

Reply:
Is there config file related to this? If so, could you point to it? Personally I very much dislike this change, and feel like crafting setups take unnecessairly big space, would like to know if I can change the setting on my server. Thanks in advance!

Reply:
https://github.com/Edivad99/ExtraStorage/commit/409d7d8126708e0c141c6be6eb8fce313c1fe6b8